convention.md

```diff
@@ -1,53 +1,6 @@
-![Homie banner](banner.png)
+# The Homie Convention
 
-<h1 align="center">The Homie Convention</h1>
-<p align="center"><i>A lightweight MQTT convention for the IoT</i></p>
-
-<p align="center">Version: <b><!--VERSION-->3.x.x<!--VERSION--></b> • <a href="https://github.com/homieiot/convention/tags">Browse Versions…</a></p>
-
-**![Rolling Release](https://cdn2.iconfinder.com/data/icons/thesquid-ink-40-free-flat-icon-pack/64/traffic-light-20.png) The Homie Convention follows the rolling release model and is improved and released in short development cycles. Check the *releases* tab to learn about previous release versions.**
-
-Implementations of the Homie convention can be found on [this page](implementations.md).
-
-----
-
-## Table of Contents
```

Comment:
Personally i still think the table of contents is handy. Usefully for when you want to find a cretin bit in the convention/

Reply:
The ToC is kind of hard to maintain. It's available on the website, and it can be automatically generated on the `master` version, anyway. 😉

Reply:
Even better :)